graspologic/plot/plot.py

```diff
@@ -673,6 +676,220 @@ def pairplot(
     return pairs
 
 
+def _plot_ellipse(
+    data, X, j, k, means, covariances, ax, label_palette, cluster_palette, alpha
+):
+    r"""
+    plot_ellipse makes a scatter plot from the two dimensions j,k where j
+    corresponds to x-axis
+    and k corresponds to the y-axis onto the axis that is ax. plot_ellipse then
+    applies a gmm ellipse onto the scatterplot
+    using the data from Y_, means, covariances.
```

Comment:
I see that `Y_` is defined in `pairplot_with_gmm` but not introduced here?

Reply:
The Y_ being referenced as data['clusters'] I will make that clearer.

Reply:
fixed in https://github.com/microsoft/graspologic/pull/541/commits/32bc140f8526a29691b5d7bcf57ceb048316b900